panel/src/components/Layout/Card.vue

```diff
@@ -19,12 +19,18 @@
     </component>
 
     <nav class="k-card-options">
-      <k-status-icon
-        v-if="flag"
-        v-bind="flag"
-        class="k-card-options-button"
-      />
       <slot name="options">
+        <k-button
+          v-if="flag"
+          v-bind="flag"
+          class="k-card-options-button"
+          @click="flag.click"
+        />
+        <k-status-icon
+          v-else-if="statusIcon"
+          v-bind="statusIcon"
+          class="k-card-options-button"
+        />
```

Comment:
I don't know about the inner workings of these components, but in `Card` (unlike `ListItem`), the components *were* outside the options slot before. However I can imagine that it is correct like you did it now.

Reply:
Yes, I think it was wrong before. The idea of the slot is that you can replace the options/flag entirely with your own buttons. I.e. 

```
<k-list-item>
  <template slot="options">
    <my-own-flag />
    <my-own-dropdown />
  </template>
</k-list-item>
```

Reply:
Yep, that's why I thought as well. 👍